src/main/java/org/folio/processing/mapping/mapper/reader/record/marc/MarcRecordReader.java

```diff
@@ -250,7 +250,9 @@ private boolean equalsBasedOnBrackets(String ruleName, String mappingParameter,
         return true;
       } else if (retrieveNameWithoutBrackets(mappingParameter).equalsIgnoreCase(value)) {
         return true;
-      } else if (ruleName.equalsIgnoreCase(VENDOR_ID) && retrieveNameWithoutBrackets(mappingParameter).equalsIgnoreCase(retrieveNameWithoutBrackets(value))) {
+      } else if (ruleName.equalsIgnoreCase(VENDOR_ID) &&
+        (retrieveCodeWithoutBrackets(mappingParameter).equalsIgnoreCase(retrieveNameWithoutBrackets(value)) ||
```

Comment:
Hi @VRohach! could you tell me please, shouldn't we compare code with code here? instead of code with name?

Reply:
Hello @JavokhirAbdullayev! There will be comparing code for vendor from  organizations and value from mapping profile. I don't know how to rename "retrieveNameWithoutBrackets"-method because it was already using for retrieving name for vendor from organizations.

Reply:
@JavokhirAbdullayev  Btw I renamed it to "retrieveNameOrValueWithoutBrackets".